package.json

```diff
@@ -113,7 +117,7 @@
     "*.hbs": "ember-template-lint --fix"
   },
   "volta": {
```

Comment:
@Windvis Do we still need it?

Reply:
I'm not sure if anyone on the team still uses Volta, but I would remove it.